### DISCUSSION TITLE
Unable to signup to the “beta build” account using the iPad Pro application

### DISCUSSION THREAD

**Discussion by sergeyu** [blinksh/blink#1739] [Support]
Hello! 
I really like the app and just updated it to the recent version (including beta build subscription).
The problem is: I’m getting an error message “unexpected error 403” (see screenshot attached)
I see no issue with ashy ssh connections.
It doesn’t seem like a temporarily issue  - I tried multiple times in last 2 days and still getting the same result.

![E3842603-1E74-4A4C-B4B1-D3CF824A83B9](https://user-images.githubusercontent.com/2042815/228338840-f868e372-d1f7-47de-bdb9-591190d17de1.jpeg)

iPad info:
![7A64F3E0-FADA-4F11-BC22-A272A54F2D17](https://user-images.githubusercontent.com/2042815/228339423-83e4c0da-7291-47f1-9701-f6fa44694bfc.jpeg)

I found no issues in my local network, so I have no idea how to troubleshoot and resolve the issue.

Has anyone met this before ?
Any ideas ?

Thanks in advance !

**Comment by carloscabanero** [comment]
Hey! Thanks for your message and sorry about that. There may be something related to your account that doesn't let you set it up. Can you reach out to us on support at blink.build? Please include your User ID too. You can get it from  config > Support > Copy User ID.

**Reply by sergeyu** [reply]
Thank you for the quick response. 
Could you please advise, how can I reach out the blind.build support ?
I’m sorry if I missing something.

**Reply by carloscabanero** [reply]
Sorry, missed your reply. Yep, that would be support@blink.sh

**Reply by carloscabanero** [reply]
Hey Sergey, please try again whenever you get a chance. Just fixed an issue that may have been affecting a few users during Sign Up, and I think this may be your problem. Let me know!
